tools/migrate-db/main.go

```diff
@@ -0,0 +1,396 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/cosmos/cosmos-db"
+)
+
+func main() {
+	homeDir := flag.String("home", os.ExpandEnv("$HOME/.celestia-app"), "Node home directory")
+	dryRun := flag.Bool("dry-run", false, "Run migration in dry-run mode without making changes")
+	noBackup := flag.Bool("no-backup", false, "Skip creating backup of data directory before migration")
+	cleanup := flag.Bool("cleanup", false, "Remove old LevelDB files after successful migration (not recommended)")
```

Comment:
looks like this isn't specified in the usage command and doesn't change behavior so it should be removed.